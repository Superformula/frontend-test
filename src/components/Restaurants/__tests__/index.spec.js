import React from 'react';
import { shallow } from 'enzyme';
import { Restaurants } from '../';

describe('Restaurants', () => {
  it('should render without failing', () => {
    const wrapper = shallow(<Restaurants />);
    expect(wrapper).toMatchSnapshot();
  });
});
