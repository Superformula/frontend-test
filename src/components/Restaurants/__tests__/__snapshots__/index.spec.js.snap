// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Restaurants should render without failing 1`] = `
.c0 {
  display: grid;
  grid-template-columns: 1fr;
  grid-gap: 32px;
  padding-bottom: 48px;
}

@media (min-width:758px) {
  .c0 {
    padding-bottom: 80px;
    grid-template-columns: repeat(2,1fr);
    grid-gap: 80px 32px;
  }
}

@media (min-width:1200px) {
  .c0 {
    grid-template-columns: repeat(4,1fr);
  }
}

@media (min-width:1600px) {
  .c0 {
    grid-template-columns: repeat(6,1fr);
  }
}

<div
  className="c0"
/>
`;
